android-core/src/main/java/com/mparticle/internal/DeviceAttributes.java

```diff
@@ -256,7 +256,7 @@ public void updateDeviceInfo(Context context, JSONObject deviceInfo) {
                 //check instance nullability here and decline to act if it is not available. Don't want to have the case where we are overriding isLimiAdTrackingEnabled
                 //just because there was a timing issue with the singleton
                 if (instance !=  null) {
-                    if (adIdInfo.isLimitAdTrackingEnabled && instance.Internal().getConfigManager().getRestrictAAIDBasedOnLAT()) {
+                    if (adIdInfo.isLimitAdTrackingEnabled) {
```

Comment:
the message below is a little wrong now with the latest android OS, Ad ID is not "available" - the library is available, but ad tracking is disabled

Reply:
i changed the copy